Signal/src/ViewControllers/HomeViewController.m

```diff
@@ -914,10 +914,10 @@ - (void)updateMappings
 
 - (YapDatabaseConnection *)uiDatabaseConnection
 {
-    NSAssert([NSThread isMainThread], @"Must access uiDatabaseConnection on main thread!");
+    OWSAssert([NSThread isMainThread]);
```

Comment:
Applied `OWSAssert()` in a few places.

Reply:
Done.  I'm going to break this out into Swift and Obj-C versions - the Obj-C version can compile out completely in release builds.

```
// This macro is intended for use in Objective-C.
#define OWSAssertIsOnMainThread() OWSCAssert([NSThread isMainThread])

// This function is intended for use in Swift.
void AssertIsOnMainThread(void);
```